src/containers/layout/footer/Footer.styles.ts

```diff
@@ -1,19 +1,24 @@
 export const styles = {
   footer: {
     backgroundColor: 'primary.900',
-    color: 'primary.50',
-    marginTop: 'auto'
+    color: 'primary.50'
   },
   container: {
     display: 'flex',
+    position: 'relative',
     flexDirection: { xs: 'column-reverse', sm: 'row' },
     justifyContent: { xs: 'space-between', sm: 'space-around' },
     alignItems: 'center',
     py: { xs: '12px', sm: '26px' }
   },
+  title: {
+    position: { xs: 'static', sm: 'absolute' },
```

Comment:
It is not a great idea to use absolute with negative values, try to center it with flex or margin, also, notice that you have space around in container